Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.42065&attempt_id=bdcc23a2-43a7-4007-89d4-25abcc3b9532)

Attempted: python3.pkgs.pythonix

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/c31hd5m0i1hkyml99qh50p7f1p2wwyfb-nix-2.0.4-dev/include/nix/util.hh:23:9: warning: 'DT_DIR' macro redefined [-Wmacro-redefined]
#define DT_DIR 3
        ^
/nix/store/0yvai96flnxvsl8bn95nwcz1y7pd7xa5-Libsystem-osx-10.11.6/include/sys/dirent.h:126:9: note: previous definition is here
#define DT_DIR           4
        ^
3 warnings generated.
ninja: build stopped: subcommand failed.
builder for '/nix/store/d464qngb1ybqjfz1sblsm7r6q1lp5f6w-pythonix-0.1.1.drv' failed with exit code 1
error: build of '/nix/store/d464qngb1ybqjfz1sblsm7r6q1lp5f6w-pythonix-0.1.1.drv' failed
```
</p></details>

--- a/pkgs/development/python-modules/pythonix/default.nix
+++ b/pkgs/development/python-modules/pythonix/default.nix
@@ -1,21 +1,26 @@
-{ stdenv, fetchFromGitHub, ninja, meson, pkgconfig, nixUnstable, isPy3k }:
+{ stdenv, fetchFromGitHub, ninja, boost, meson, pkgconfig, nix, isPy3k }:
 
-assert isPy3k;
 
 stdenv.mkDerivation rec {
   name = "pythonix-${version}";
-  version = "0.1.0";
+  version = "0.1.4";
 
   src = fetchFromGitHub {
     owner = "Mic92";
     repo = "pythonix";
     rev = "v${version}";
-    sha256 = "1piblysypyr442a6najk4mdh87xc377i2fdbfw6fr569z60mnnnj";
+    sha256 = "1q1fagfwzvmcm1n3a0liay7m5krazmhw9l001m90rrz2x7vrsqwk";
   };
 
-  nativeBuildInputs = [ meson pkgconfig ninja ];
+  disabled = !isPy3k;
 
-  buildInputs = [ nixUnstable ];
+  nativeBuildInputs = [ meson ninja pkgconfig ];
+
+  buildInputs = [ nix boost ];
+
+  checkPhase = ''
+    ninja test
+  '';
 
   meta = with stdenv.lib; {
     description = ''
@@ -23,6 +28,5 @@ stdenv.mkDerivation rec {
     '';
     maintainers = [ maintainers.mic92 ];
     license = licenses.mit;
-    platforms = platforms.linux;
   };
 }

